content/hardware/03.nano/boards/nano-matter/tutorials/08.ml-magic-wand/content.md

```diff
@@ -0,0 +1,663 @@
+---
+title: 'ML Magic Wand with the Arduino Nano Matter'
+difficulty: beginner
+compatible-products: [nano-matter]
+description: 'Learn how to build a Gesture Recognition Magic Wand using the Arduino Nano Matter'
+tags:
+  - AI
+  - ML
+  - IMU
+  - Modulino
+  - Matter
+
+author: 'Christopher Méndez'
+hardware:
+  - hardware/03.nano/boards/nano-matter
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+  - iot-cloud
+---
+
+## Overview
+
+This tutorial describes how to build a gesture recognition system based on a machine learning model using TensorFlow and the Arduino Nano Matter.
+
+![Nano Matter Magic Wand](assets/thumbnail.png)
+
+The Arduino Nano Matter acts as a digital magic wand 🪄, where sensor data from its movements is processed by a model to classify and detect specific gestures. The inference results will turn on a LEDs array on a specific color respectively to the detected gesture.
```

Comment:
I would change LEDs array for Modulino Pixels, stating the official name of the Arduino products.